include/coro.h

```diff
@@ -312,11 +312,11 @@ void coro_stack_free (struct coro_stack *stack);
     && !defined CORO_SJLJ    && !defined CORO_LINUX \
     && !defined CORO_IRIX    && !defined CORO_ASM \
     && !defined CORO_PTHREAD && !defined CORO_FIBER
-# if defined WINDOWS && (defined __i386__ || (__x86_64__ || defined _M_IX86 || defined _M_AMD64)
+# if defined WINDOWS && (defined __i386__ || (defined __x86_64__ || defined _M_IX86 || defined _M_AMD64))
```

Comment:
The parentheses are grouping an or-only expression (i.e. `||`), so wouldn't it be simpler to just write the expression in the parentheses as `(defined __i386__ || defined __x86_64__ || defined _M_IX86 || defined _M_AMD64)`; that is, we'd remove the inner parentheses.